docs/contributors/code/release.md

```diff
@@ -50,22 +50,44 @@ Once released, all that's left to do is writing a release post on [make.wordpres
 -   If subtasks of a related set of pull requests are substantial, consider organizing as entries in a nested list.
+-   If one or more pull requests revert one or more pull requests in the same release netting a zero-sum of code changes, remove them.
 -   Remove mobile app pull request entries.
 
-Once you have cleaned up the changelog, choose a few features to highlight in the release post and record an animation of them in use.
+#### 2. Selecting the release highlights
+Once you have cleaned up the changelog, the next step is to choose a few changes to highlight in the release post. These highlights usually focus on new features and enhancements, including performance and accessibility ones, but can also include important API changes or critical bug fixes.
 
-You should also include a performance audit at the end of the release post. You can use `bin/plugin/cli.js perf` to automatically do this.
+Given the big scope of Gutenberg and the high number of pull requests merged in each milestone, it is not uncommon to overlook impactful changes worth highlighting; it is recommended to make this step a collaborative effort and share your picks with the Gutenberg Core team.
 
-Compile this to a draft post on [make.wordpress.org/core](https://make.wordpress.org/core/); this post should be published after the actual release.
+#### 3. Drafting the release post
+
+When possible, the highlighted changes in the release post should include an animation (video or GIF) or a static image of them in use. Choosing between an animation or a static image will depend on the best way to showcase each highlight: while animations are better to demonstrate workflows, more direct highlights can be shown with an image; having too many video players adds a layer of friction for users to see the highlight, whereas too many simultaneous animated GIFs can distract and overwhelm the reader.
+
+These visual assets should maintain consistency with previous release posts; using lean, white themes helps in this regard and visually integrate well with the [make.wordpress.org/core](https://make.wordpress.org/core/) blog aesthetics. Including copyrighted material should be avoided, and browser plugins that can be seen in the browser canvas (spell checkers, form fillers, etc.) disabled when capturing the assets. If you would like help creating the visual assets, please ping somebody in the Gutenberg Core team in advance so that it can be planned for.
+
+#### 4. Running the performance tests
+
+Your post should also include a performance audit at the end, comparing the current Gutenberg release with both the previous one and the latest WordPress major version. You can use `bin/plugin/cli.js perf` to automatically do this for you by passing as parameters the tags to run the performance suite against, such as `bin/plugin/cli.js perf release/x.y release/x.z wp/a.b`
```

Comment:
I think now we have a job that is run on CI that gives you the results, so there's no need to run this on your own in theory.

Reply:
I think @mcsf updated the job to use the latest WP release as base branch which should in theory make it more stable.

Reply:
Indeed, I failed to add that it should be fixed in 10.8 thanks to #32244 👍